test/ffmpeg-vaapi/encode/encoder.py

```diff
@@ -90,12 +90,22 @@ def gen_name(self):
       name += "-{loopshp}"
     if vars(self).get("looplvl", None) is not None:
       name += "-{looplvl}"
+    if vars(self).get("r2r", None) is not None:
+      name += "-r2r"
 
     return name.format(**vars(self))
 
   def before(self):
     self.refctx = []
 
+  def gen_output(self):
```

Comment:
`def call_ffmpeg(self, iopts, oopts)):` probably better.

Reply:
changed to call_ffmpeg(self, iopts, oopts)):